src/Symfony/Component/Console/Descriptor/TextDescriptor.php

```diff
@@ -140,6 +140,13 @@ protected function describeCommand(Command $command, array $options = array())
         $command->getSynopsis(false);
         $command->mergeApplicationDefinition(false);
 
+        if ($description = $command->getDescription()) {
+            $this->writeText('<comment>Description:</comment>', $options);
+            $this->writeText("\n");
+            $this->writeText('  '.$description);
+            $this->writeText("\n\n");
+        }
+
```

Comment:
should as well the ```$command->getProcessedHelp()``` return only the ```$command->getHelp()``` and not sometimes the ```$command->getDescription()```?

Reply:
I think it should yes, it's weird that `getProcessedHelp()` can return the description., but I'm not sure it's fine to change it now. Instead, could we display the `Help` section (calling `getProcessedHelp()`) only if `getHelp()` is not null? or only if `getProcessedHelp() !== getDescription()`?

Reply:
@chalasr ok, I will change the called method here